cli/commands.go

```diff
@@ -76,3 +91,40 @@ var get = &cobra.Command{
 		return nil
 	},
 }
+
+var server = &cobra.Command{
+	Use:   "server",
+	Short: "starts server",
+	Long:  "give 2 parameters: port and filepath",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		cReader := &constReader{}
+		port, err := cmd.Flags().GetString("port")
+		if err != nil {
+			return err
+		}
+		path, err := cmd.Flags().GetString("path")
+		if err != nil {
+			return err
+		}
+		secretService := service.New(storage.NewFsStorage(path),
+			ciphering.NewAESEncoder(ciphering.NewRandomNonceProducer(rand.Reader)),
+			ciphering.NewAESEncoder(ciphering.NewRandomNonceProducer(cReader)))
+
+		sign := make(chan os.Signal, 1)
+		signal.Notify(sign, syscall.SIGHUP, syscall.SIGINT, syscall.SIGTERM, syscall.SIGQUIT)
```

Comment:
you can move it inside goroutine since it is only used inside its scope. (line 119)